src/components/accordion/Accordions.tsx

```diff
@@ -0,0 +1,98 @@
+import { FC, ReactNode } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import Typography from '@mui/material/Typography'
+import Box from '@mui/material/Box'
+import Accordion, { AccordionProps } from '@mui/material/Accordion'
+import AccordionSummary from '@mui/material/AccordionSummary'
+import AccordionDetails from '@mui/material/AccordionDetails'
+
+import { styles } from '~/components/accordion/Accordion.styles'
+
+import { AccordionItem } from '~/types'
+import { Variant } from '@mui/material/styles/createTypography'
+
+interface Sx {
+  [key: string]: any
+}
+
+interface AccordionsProps
+  extends Omit<AccordionProps, 'onChange' | 'children'> {
+  items: AccordionItem[]
+  onChange: (value: number) => void
+  activeIndex: number[] | number | null
+  multiple?: boolean
+  icon?: ReactNode
+  sx?: Sx
+  titleVariant: Variant
+  descriptionVariant: Variant
+}
+
+const Accordions: FC<AccordionsProps> = ({
+  items,
+  onChange,
+  activeIndex,
+  multiple = false,
```

Comment:
we can omit this prop by adding variable multiple = Array.isArray(activeIndex)

Reply:
refactored